src/EnergyPlus/WindowManager.cc

```diff
@@ -316,16 +325,29 @@ namespace WindowManager {
         Real64 ShadeReflFacVis; // Shade/blind visible reflection factor
         int SpecDataNum;        // Spectral data set number
         int numptDAT;           // Number of wavelengths in a spectral data set
-        int ISlatAng;           // Slat angle counter
         bool StormWinConst;     // True if a construction with a storm window
         bool Triangle;          // True if window is triangular
         bool Rectangle;         // True if window is rectangular
-        Array1D<Real64> W1(3);  // Window vertices (m)
-        Array1D<Real64> W2(3);
-        Array1D<Real64> W3(3);
-        Array1D<Real64> W21(3); // W1-W2, W3-W2, resp. (m)
-        Array1D<Real64> W23(3);
-
+        Vector3<Real64> W1;  // Window vertices (m)
```

Comment:
Use the `Vector3` class rather than the more general `Array1D` for 3D geometry calcs. Many advantages.